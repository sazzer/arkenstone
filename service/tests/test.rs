#![cfg_attr(
    feature = "cargo-clippy",
    allow(
        clippy::module_name_repetitions,
        clippy::module_inception,
        clippy::used_underscore_binding,
        clippy::wildcard_imports,
        clippy::missing_errors_doc
    )
)]

mod authentication;
mod health;
pub(crate) mod service;
