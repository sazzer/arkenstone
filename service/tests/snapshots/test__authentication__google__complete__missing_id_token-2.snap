---
source: tests/test.rs
expression: res.to_json().unwrap()
---
{
  "detail": "Authentication failed with the authentication provider: ID Token was not present in response",
  "status": 400,
  "title": "Failed to authenticate user",
  "type": "tag:arkenstone,2020:authentication/problems/authentication_error"
}
